test/stressRooStats_tests.h

```diff
@@ -1475,6 +1475,10 @@ class TestHypoTestInverter1 : public RooUnitTest {
       return kTRUE;
    }
 
+   // larger value test tolerance especially when using toys (difference of <~ 0.1 observed between using Minuit or Minuit2)
```

Comment:
Which one is the better minimum, that of Minuit or Minuit2? If the latter - should that regression get fixed rather than hiding it behind a larger tolerance?

Reply:
There is no in general a better minimum. The differences are simply due to numerical errors, which make the convergence of the two minimiser slightly different. The level is much smaller than 0.1, that tolerance is on the RooStats results which can be different due to different toys. A small difference in the minimizer results could create different toys and then a more different result.